Emoji picker

tfudoinkebs opened the pull request:
**Added Emoji Picker to Post & Comment Section:**
![image](https://github.com/user-attachments/assets/bf5df264-22dc-432e-92b0-13fe4a618952)
![image](https://github.com/user-attachments/assets/ea808610-ef6a-476a-9918-9e790345b10e)

**Issues:**
- The post and comment forms don't instantly recognize emojis as valid input. You have to click the "Post" or "Comment" button for the button to become active.
- Poor loading implementation when the emoji picker is clicked.
- The emoji picker doesn't load immediately when clicked.

kevinroinuesca-coder committed:
fixed private route on donate, about, & FAQ pages

kevinroinuesca-coder committed:
added emoji picker

kevinroinuesca-coder committed:
added themes on emoji picker

kevinroinuesca-coder committed:
added emoji picker on comment input

kevinroinuesca-coder committed:
fixed layout post-page emoji picker

kevinroinuesca-coder committed:
sticky to fixed comment section

GabrielAstilla reviewed (commented):
I think this PR is good to be deployed in the staging environment. This also fixed the bug on profile pictures on the comment section.

kevinroinuesca-coder committed:
Merge branch 'dev' of https://github.com/bricesuazo/kabsu.me into feat/emoji-picker

kevinroinuesca-coder committed:
- changed emoji-oicker-react to emoji-mart
- changed the maximum characters in MentionsInput to 512
- removed loader on emoji picker when clicked

kevinroinuesca-coder committed:
removed unused packages
